numpy/polynomial/polynomial.py

```diff
@@ -1277,6 +1288,11 @@ def polyfit(x, y, deg, rcond=None, full=False, w=None):
 
         For more details, see `numpy.linalg.lstsq`.
 
+    covariance_matrix : ndarray
+        This ndarray is only returned if ``cov == True``
```

Comment:
This is not quite right as written. It is also true if `cov == 'unscaled'`. But see my comment above about how I dislike this call pattern.